CONTRIBUTING.adoc

```diff
@@ -246,6 +246,27 @@ Be aware that some of this content might already be present in the Jenkins User
 
 The documentation pages can use the same metadata (`title`, `description`, `opengraph:image`) as blog posts.
 
+* When creating or adding documentation, make sure that every numbered line contains a complete sentence, including an ending punctuation mark (`.`, `?`, `!`).
```

Comment:
Unclear. What's a numbered line? Are there exceptions? What about line 123 ending without punctuation, or line 248 having no content?

Reply:
Good point on this @daniel-beck, I've reworded this so that it is more clear and provides a reference for the style.